submissions/career/career.md

```diff
@@ -0,0 +1,53 @@
+
+![Sky Factory Auto Sieve + Processing Setup]()
+*Here, cobblestone is being mined automagically, then crushed into gravel before being sieved and smeling into precious, precious ores.*
+
+The idea that I could make a thing that does my job for me was amazing. Anyone who's played Factorio will of course recognize this idea as the key goal in the game - **Automate *EVERYTHING***. After spending far, far too much time in Sky Factory, I expanded my horizons to other games but found the types of games my friends we're into (FPSes primarily), just weren't my cup of tea. Out in the distance haze, Factorio beckoned me, and I of course bought the game, excited at the possibilities it provided me. 
+
+Unfortunately, it wasn't all sunshine and roses in my journey into the depths of Factorio. As is not uncommon, I gleefully enter the game, made and automated the red and green packs then burned out. Looking back, I suspect there was some Complexity Bias going on here, which again is very common in Factorio if you aren't prepared to break down problems and understand their individual components. The game sat by the wayside, I tried other games and left poor Factorio out in the cold, not a friend in the world.
+
+Several months later, my cousin came to visit and saw I had Factorio - he was excited to play it so we decided that together we would give it a go. After some sketchy library sharing (yay Steam offline mode), we spent a weekend of bliss together, figuring out how the game worked and progressing past the ever challenging blue science packs. *Side note: This is a really great example of Piracy for good since that weekend of Factorio caused my cousin to go home and buy the game himself so we could keep playing together, something that might not have happened if not for our dodgy dealings.* This push past the percieved complexity forever ruined me, it caused the crippling Cracktorio addiction that many of us share, meaning I never could put this beast back down.
+
+So... why am I telling you this? *Fantastic question, really glad you asked. Very well asked question I might add.* For me it's pretty clear that playing modded Minecraft and Factorio pushed forward my critical thinking and mathematical skills and diverted me down the STEM path of goodness. Now, I'm studying Robotic Engineering and though Factorio isn't the only thing that pushed me down this path, it definitely helped guide me to where I am now.
+
+### The computer science angle
```

Comment:
A few other ideas for expansion:

- Factorio has created fairly large community software projects that pushed people to learn more than simply coding, etc. Clusterio, Factorio Mod Debugger, etc. Modding is a gateway drug to coding.
- Therenas -> forced to start coding -> learned and mastered -> started wikies etc -> now part of Wube for that exact purpose
- Simple mods are really easy to create and theres many tutorials to follow. But, just like Factorio itself, you can dive super deep into complexity and crazy things - if you want to.
- Combinators are similar to what you learn first in your first programming course. When I started with programming at 16 we played with Excel formulas. The goal was to learn "I need this thing, but how do I get it? What is each step to take to get there? " Factorio even teaches this to you without combinators. I want to a furnace array. For that I need create a mine, belts, and the furnaces. For the mine I need more power and kill those biters. I need a belt factory... It's that same endless loop that hooks you to simply keep on playing. I